test/combinations.jl

```diff
@@ -2,6 +2,56 @@
 # - All supported combinations of integral(f, ::Geometry, ::IntegrationAlgorithm) produce accurate results
 # - Invalid applications of integral aliases (e.g. lineintegral) produce a descriptive error
 
+@testitem "Meshes.Ball 2D" setup=[Setup] begin
+    origin = Point(0, 0)
+    ball = Ball(origin, 2.8)
+
+    f = p -> one(Float64)
```

Comment:
```suggestion
    f(p) = 1.0
```
I think this notation easier to read.